api/v1alpha1/openapi.yaml

```diff
@@ -4620,6 +4620,8 @@ components:
           description: 'Array of resource monitor configurations.'
           items:
             $ref: '#/components/schemas/ResourceMonitor'
+        decommissionRequested:
```

Comment:
It'd have to be "decommision**ing**Requested" (noun, not verb), or?
However, the user's intent would be to have the device decommissioned/wiped/..., not to request something. That intent is in the "target" of the decom schema, so maybe a grouping noun like "decommissioning" would be more appropriate?